extras/indexing/src/test/java/org/apache/rya/indexing/mongo/MongoFreeTextIndexerTest.java

```diff
@@ -43,27 +44,36 @@
 import org.openrdf.model.vocabulary.RDFS;
 
 import com.google.common.collect.Sets;
+import com.mongodb.MongoClient;
 
 import info.aduna.iteration.CloseableIteration;
 
-public class MongoFreeTextIndexerTest extends MongoRyaTestBase {
+public class MongoFreeTextIndexerTest {
```

Comment:
Well from my perspective it appears to be more than a unit test because it integrates with MongoDB, and that runs as a separate process.  I think that for this to be considered a unit test, it would have to make use of a Mock instead of MongoDB.